CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Replace `caip-x` with `caip-348` in Multichain API over externally_connectable ([#32070](https://github.com/MetaMask/metamask-extension/pull/32070))
 
 ### Changed
+- Updated institutional snap ([#32208](https://github.com/MetaMask/metamask-extension/pull/32208))
```

Comment:
Should this be in fix? And also i believe it will be helpful to include the message containing what is fixed, eg: quote from the PR description `to fix issues with dev mode that introduce unwanted behaviour`, wdyt?

Reply:
Many thanks 🙏 it is updated now